tests/unit/ext/switches/brocade.py

```diff
@@ -20,77 +20,77 @@
 from haas.ext.obm.ipmi import Ipmi
 
 MODE_RESPONSE_ACCESS = """
```

Comment:
as far as I can tell there is not, I think we can ignore a file but I'd wager that is a slippery slope